### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by uma-kt**
# Description

Fixes #

## How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

## Checklist:

* [ ] The title of the PR states what changed and the related issues number (used for the release note).
* [ ] Does this PR requires documentation updates?
* [ ] I've updated documentation as required by this PR.
* [ ] I have performed a self-review of my own code.
* [ ] I have commented my code, particularly in hard-to-understand areas.
* [ ] I have tested it for all user roles.
* [ ] I have added all the required unit test cases.

## Does this PR introduce a breaking change for other components like worker-operator?

```release-note

```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,70 +45,73 @@ helm repo update
...
-````
+      controller:
+      ---
+      ---
+         image: <my-custom-image> 
+         tag: <unique-tag>
+   ```
 
-2. Deploy the updated chart.
+2. Deploy the updated chart
```
Comment (veenamj-avesha) [inline README.md:114]: The sentence should have a period. Why did you remove it?

File: README.md [hunk 2]
```diff
@@ -45,70 +45,73 @@ helm repo update
...
-git clone https://github.com/kubeslice/kubeslice-controller.git
-cd kubeslice-controller
-```
+   ```console
+   git clone https://github.com/kubeslice/kubeslice-controller.git
+   cd kubeslice-controller
+   ```
 
 2. Adjust image name variable `IMG` in the [`Makefile`](Makefile) to change the docker tag to be built.
    Default image is set as `IMG ?= aveshasystems/kubeslice-controller:latest`. Modify this if required.
```
Comment (veenamj-avesha) [inline README.md:54]: The default image is...

File: README.md [hunk 3]
```diff
@@ -45,70 +45,73 @@ helm repo update
...
-make docker-build
-```
-### Running Local Image on Kind Clusters
+   ```console
+   make docker-build
+   ```
+
+### Run Local Image on Kind Clusters
 
 1. Loading kubeslice-controller image into your kind cluster ([kind](https://kind.sigs.k8s.io/docs/user/quick-start/#loading-an-image-into-your-cluster)).
```
Comment (veenamj-avesha) [inline README.md:62]: Load the `kubeslice-controller` image ...

File: README.md [hunk 4]
```diff
@@ -45,70 +45,73 @@ helm repo update
...
+   Example
 
-```console
-kind load docker-image aveshasystems/kubeslice-controller --name kind
-```
+   ```console
+   kind load docker-image aveshasystems/kubeslice-controller --name kind
+   ```
 
 2. Check the loaded image in the cluster. Modify node name if required.
```
Comment (veenamj-avesha) [inline README.md:76]: Modify the node name ...

File: README.md [hunk 5]
```diff
@@ -45,70 +45,73 @@ helm repo update
...
-
-* Note: `kind-control-plane` is the name of the Docker container. Modify if needed.
```
Comment (veenamj-avesha) [inline README.md:77]: Modify the name if needed.

File: README.md [hunk 6]
```diff
@@ -45,70 +45,73 @@ helm repo update
...
+   
+   :::note
+  `kind-control-plane` is the name of the Docker container. Modify if needed.
```
Comment (veenamj-avesha) [inline README.md:79]: Modify the name if needed.

File: README.md [hunk 7]
```diff
@@ -45,70 +45,73 @@ helm repo update
...
+  :::
 
 ```console
 docker exec -it kind-control-plane crictl images
 ```
-### Deploying in a Cluster
+### Deploy on a Cluster
 1. Create chart values file `yourvaluesfile.yaml`. Refer to [values.yaml](https://github.com/kubeslice/charts/blob/master/charts/kubeslice-controller/values.yaml) on how to adjust this and update the `kubeslice-controller` image to the local build image.
```
Comment (veenamj-avesha) [inline README.md:86]: can you replace adjust with set the values or modify/edit or something?

File: README.md [hunk 8]
```diff
@@ -45,70 +45,73 @@ helm repo update
...
-```console
-git clone https://github.com/kubeslice/kubeslice-controller.git
-cd kubeslice-controller
-```
+   ```console
+   git clone https://github.com/kubeslice/kubeslice-controller.git
+   cd kubeslice-controller
+   ```
 
 2. Adjust image name variable `IMG` in the [`Makefile`](Makefile) to change the docker tag to be built.
```
Comment (veenamj-avesha) [inline README.md:53]: Modify  the image name variable... 

or you can start replace Adjust with Edit...

**Review by veenamj-avesha — COMMENTED** [review]
I have a few comments. Please incorporate the feedback.

File: README.md (hunk 2 above)
Comment (uma-kt) [reply README.md]: done

File: README.md (hunk 4 above)
Comment (uma-kt) [reply README.md]: done

File: README.md (hunk 6 above)
Comment (uma-kt) [reply README.md]: done

File: README.md (hunk 8 above)
Comment (uma-kt) [reply README.md]: done

**Review by veenamj-avesha — APPROVED** [review]
LGTM
